### PR TITLE
MLIBZ-2685: Cirrus CI integration

### PR DESCRIPTION
**Pull request by heyzooi**
#### Description

Cirrus CI integration

#### Changes

- `.cirrus.yml`

#### Tests

- Same unit tests

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  test_script: cd Kinvey.Tests; dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
+
+task:
+  name: macOS Mono
+  osx_instance:
+    image: high-sierra-xcode-9.4.1
+  env:
+    PATH: $PATH:/Library/Frameworks/Mono.framework/Versions/Current/Commands
+  install_mono_sdk_script: brew cask install mono-mdk
+  check_version_script: mono --version
```
Comment (vinaygahlawat) [inline .cirrus.yml]: Is there a way to run `Kinvey.Tests` for the macOS Mono task?

File: Kinvey/Kinvey.csproj [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="SQLite.Net-PCL" Version="3.1.1" />
     <PackageReference Include="SQLite.Net.Async-PCL" Version="3.1.1" />
     <PackageReference Include="Remotion.Linq" Version="2.2.0" />
-    <PackageReference Include="Pubnub" Version="4.0.21" />
+    <PackageReference Include="Pubnub" Version="4.0.22" />
```
Comment (vinaygahlawat) [inline Kinvey/Kinvey.csproj]: Why does the `Pubnub` version have to be bumped up here?

Comment (heyzooi) [reply Kinvey/Kinvey.csproj]: I got a crash because of PubNub...

https://cirrus-ci.com/task/4644658730762240

**Comment by heyzooi** [comment]
@vinaygahlawat I think you can approve this PR and we can fix the issue on Windows later. What do you think?

**Comment by heyzooi** [comment]
@vinaygahlawat could you approve this again please

**Pull request merged**